book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] Great job, here we need both setters and getters for rentals. Kindly implement that.